modules/resetdigitalBidAdapter.js

```diff
@@ -43,16 +43,29 @@ export const spec = {
       };
     }
 
+    if (bidderRequest && bidderRequest.uspConsent) {
+      payload.ccpa = bidderRequest.uspConsent;
+    }
+
     for (let x = 0; x < validBidRequests.length; x++) {
       let req = validBidRequests[x]
 
       payload.imps.push({
         pub_id: req.params.pubId,
+        site_id: req.params.siteID ? req.params.siteID : null,
+        placement_id: req.params.placement ? req.params.placement : null,
+        position: req.params.position ? req.params.position : null,
+        bid_floor: req.params.bidFloor ? req.params.bidFloor : null,
```

Comment:
You must also support the mostly

Reply:
Hi, can you give me an example or a link?  I don't know what you mean by the module, I appreciate any help thanks

Reply:
sure, if you take a floor on params, you must also take one in the bidrequest object. Here is a good example https://github.com/prebid/Prebid.js/pull/9279/files

Reply:
good! that really helped me, thanks!

Reply:
fixed